examples/quadrupole_softedge/analysis_quadrupole_softedge.py

```diff
@@ -20,42 +18,32 @@ def get_moments(beam):
     -------
     sigx, sigy, sigt, emittance_x, emittance_y, emittance_t
     """
-    sigx = moment(beam["x"], moment=2) ** 0.5  # variance -> std dev.
-    sigpx = moment(beam["px"], moment=2) ** 0.5
-    sigy = moment(beam["y"], moment=2) ** 0.5
-    sigpy = moment(beam["py"], moment=2) ** 0.5
-    sigt = moment(beam["t"], moment=2) ** 0.5
-    sigpt = moment(beam["pt"], moment=2) ** 0.5
+    sigx = moment(beam["position_x"], moment=2) ** 0.5  # variance -> std dev.
+    sigpx = moment(beam["momentum_x"], moment=2) ** 0.5
+    sigy = moment(beam["position_y"], moment=2) ** 0.5
+    sigpy = moment(beam["momentum_y"], moment=2) ** 0.5
+    sigt = moment(beam["position_ct"], moment=2) ** 0.5
+    sigpt = moment(beam["momentum_t"], moment=2) ** 0.5
```

Comment:
Once we have implemented the reduced diagnostics, writing the beam moments to an output file, most of the tedious analysis and duplication can be eliminated.

Reply:
Agreed!

I added a couple of future ideas to the PR description at the end. One of them would be to generate all this meta data in situ and add it to the openPMD monitor. In addition to that, we can add an option to disable the actual particle (heavy) data in the monitor and only write the meta-data as attributes.